todo-app/src/TasksDB.js

```diff
@@ -0,0 +1,17 @@
+export default [
+    {
+        text: "* Get out of bed",
+        done: "(done)"
```

Comment:
For the `done` property you could have used a boolean `true` or `false` value.

Reply:
I'll try it today